extensions/vscode/package.json

```diff
@@ -160,12 +160,13 @@
     "@types/mocha": "^10.0.6",
     "@types/node": "20.12.7",
     "@types/vscode": "^1.79.0",
-    "@typescript-eslint/eslint-plugin": "^7.0.0",
-    "@typescript-eslint/parser": "^6.21.0",
+    "@typescript-eslint/eslint-plugin": "^7.7.1",
+    "@typescript-eslint/parser": "^7.7.1",
     "@vscode/test-electron": "^2.3.9",
     "c8": "^9.1.0",
     "eslint": "^8.57.0",
     "glob": "^10.3.12",
+    "minimatch": "5.1.2",
```

Comment:
We should see if we can use the [overrides](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#overrides) field to do this instead.